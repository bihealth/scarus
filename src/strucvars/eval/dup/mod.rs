//! Evaluation of duplications, gain of copy number.

pub mod result;
