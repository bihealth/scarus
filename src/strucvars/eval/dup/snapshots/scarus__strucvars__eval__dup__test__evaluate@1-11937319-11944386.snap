---
source: src/strucvars/eval/dup/mod.rs
expression: result
---
- g1:
    g1b: {}
- g3:
    g3a:
      suggested_score: 0
      num_genes: 0
      genes: []

